reconcile/utils/oc.py

```diff
@@ -63,6 +64,55 @@ class JobNotRunningError(Exception):
 
 
 class OC:
+    class _Decorators:
+        @classmethod
+        def process_reconcile_time(cls, function):
+
+            @wraps(function)
+            def wrapper(*args, **kwargs):
+                result = function(*args, **kwargs)
+
+                running_state = RunningState()
+                commit_time = float(running_state.timestamp)
+                time_spent = time.time() - commit_time
+
+                try:
+                    kind = result['resource']['kind']
+                    name = result['resource']['metadata']['name']
+                    annotations = \
+                        result['resource']['metadata'].get('annotations')
+                except KeyError as e:
+                    logging.warning(f"Error processing metric: {e}")
+                    return
+
+                ignore_reconcile_time = \
+                    annotations.get('qontract.ignore_reconcile_time') == 'true'
+                if not ignore_reconcile_time:
+                    name = f'{function.__module__}.{function.__qualname__}'
+                    reconcile_time.labels(
+                        name=name,
+                        integration=running_state.integration
+                    ).observe(amount=time_spent)
+
+                if not result['is_log_slow_oc_reconcile']:
+                    return
+
+                if time_spent > result['slow_oc_reconcile_threshold']:
```

Comment:
`slow_oc_reconcile_threshold` might not be there.

Reply:
Fixed in 69576f7